locales/en/moves.json

```diff
@@ -114,5 +114,8 @@
       "content": "Reference number<br><strong>{{moveReference}}</strong>"
     },
     "detail": "The move for <strong><a href=\"{{href}}\">{{name}}</a></strong> to <strong>{{location}}</strong> on <strong>{{date}}</strong> has been scheduled."
+  },
+  "details": {
+    "heading": "Move agreed"
```

Comment:
We can also do what we do with the assessment question steps and use the question as the heading, then use the visibility class on the fieldset legend.